apps/crn-frontend/src/analytics/productivity/Productivity.tsx

```diff
@@ -0,0 +1,104 @@
+import { FC } from 'react';
+import { useHistory, useParams } from 'react-router-dom';
+import { AnalyticsProductivityPageBody } from '@asap-hub/react-components';
+import { UserProductivityMetric } from '@asap-hub/react-components/src/organisms/UserProductivityTable';
+import { TeamProductivityMetric } from '@asap-hub/react-components/src/organisms/TeamProductivityTable';
+import { analytics } from '@asap-hub/routing';
+
+import { usePagination, usePaginationParams } from '../../hooks';
+
+const Productivity: FC<Record<string, never>> = () => {
```

Comment:
I wonder if we could type as 

```suggestion
const Productivity: FC = () => {
```

Since it seems this component does not receive any props